### ISSUE TITLE
Layer selection stays empty when opening it on the side

### ISSUE THREAD

**Issue by joshinils** [OSM-de/osm-org-greasemonkey#26]
disabled:
![image](https://github.com/OSM-de/osm-org-greasemonkey/assets/34302971/680e9a32-52ff-4664-8297-a9a4f78e649c)

with enabled script:
![image](https://github.com/OSM-de/osm-org-greasemonkey/assets/34302971/05c1f6d9-2937-4fa1-9bc3-d9c85d7bce61)

and there is this message in my console:
```
Uncaught TypeError: r.href is undefined
    updateLinks https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:231
    each https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:11
    each https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:11
    updateLinks https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:231
    <anonymous> https://www.openstreetmap.org/assets/index-68e64d2fb2dafb4ee64e783697166c9b7f646f4aea5e8959e4eba358d63669c1.js:1
    fire https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:209
    invalidateSize https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:209
    togglePane https://www.openstreetmap.org/assets/index-68e64d2fb2dafb4ee64e783697166c9b7f646f4aea5e8959e4eba358d63669c1.js:1
    i https://www.openstreetmap.org/assets/index-68e64d2fb2dafb4ee64e783697166c9b7f646f4aea5e8959e4eba358d63669c1.js:1
    dispatch https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:12
    handle https://www.openstreetmap.org/assets/application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:12
application-6bb3862e536cfaf6d6c6cad72a0ab511251ea1922a7d75eec38693e304c58321.js:231:2278
```

**Comment by kmpoppe** [comment]
Is this since the 0.27 version?

**Comment by joshinils** [comment]
idk if it is since, but it affects v27, yes

**Comment by kmpoppe** [comment]
Resolved with 0.29
